python/enthic/compute_stats.py

```diff
@@ -0,0 +1,211 @@
+            item['data']['status'] = "checked"
+            # Fix children sign and/or set to zero missing values if any
+            for child_name in item['children']:
+                flip_sign(item['children'][child_name])
+                set_to_zero_computed(item['children'][child_name])
+        elif abs((computed_sum - value) / value) > 100:
+            item['data']['status'] = "error"
+        else:
+            item['data']['status'] = "error"
+
+    if computed_sum != item['data']['value']:
+        item['data']['computedValue'] = computed_sum_from_computed
+
+def set_to_zero_computed(item):
+    if math.isnan(item['data']['value']) and ('computedValue' not in item['data'] or item['data']['computedValue'] == 0):
+        item['data']['value'] = 0
+        item['data']['status'] = "computed"
+        if 'children' in item:
+            for child_name in item['children']:
+                set_to_zero_computed(item['children'][child_name])
+
+def flip_sign(item):
+    if item.sign == -1:
+        item['data']['value'] = -item['data']['value']
+        item['data']['status'] = "signFlipped"
+
+def compute_annual_share_score(tree):
+    # Retrieve needed values
+    participation = tree['children']['ParticipationSalariesAuxResultats']['data']['value']
+    impot = tree['children']['ImpotsSurLesBenefices']['data']['value']
+    resultat_financier = tree['children']['ResultatExceptionnel']['data']['value']
+    resultat_exceptionnel = tree['children']['ResultatAvantImpot']['children']['ResultatFinancier']['data']['value']
+    resultat_exploitation = tree['children']['ResultatAvantImpot']['children']['ResultatExploitation']['data']['value']
+
+    # If any values unknown, cannot compute score
+    if any(math.isnan(value) for value in [participation, impot, resultat_financier, resultat_exceptionnel, resultat_exploitation]):
+        return float('nan')
+
+    # Compute score if possible
+    shared_part = participation + impot
+    resultat_capitaux = resultat_financier + resultat_exceptionnel
+    if shared_part >= 0:
+        if resultat_capitaux <= 0:
+            return shared_part / resultat_exploitation
+
+    return float('nan')
+
+def compute_salary_scores(tree):
+    charges = tree['children']['ResultatAvantImpot']['children']['ResultatExploitation']['children']['ChargesExploitation']
+
+    cotisations_sociales = charges['children']['ChargesSociales']['data']['value']
+    salaires = charges['children']['SalairesEtTraitements']['data']['value']
+
+    salary_level = float('nan')
+    salary_percent = float('nan')
+    if not any(math.isnan(value) for value in [cotisations_sociales, salaires]) and salaires > 0:
+        salary_level = cotisations_sociales / salaires
+        if not math.isnan(charges['data']['value']):
+            salary_percent = (cotisations_sociales + salaires) / charges['data']['value']
+
+    return salary_level, salary_percent
+
+def compute_company_statistics(company_data):
```

Comment:
Please add a properly restructured text docsrting documenting the code. You can use https://livesphinx.herokuapp.com/ to help formatting.